README.md

```diff
@@ -1 +1,116 @@
-# hello-react-front-end
\ No newline at end of file
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+<!-- PROJECT DESCRIPTION -->
+# 📖 Hello rails back end <a name="about-project"></a>
+**Hello rails back end** Is a react app that fetches data from a custom api made through rails and then displays greetings from that api.
+To check the React view repo click [here](https://github.com/cilfonegabriel/hello-react-front-end)
```

Comment:
- [ ] It would be better to update the link here to the backend repository link so users can easily navigate to the backend repository 👍🏿